src/backend/machobj.c

```diff
@@ -2861,10 +2861,6 @@ symbol *Obj::tlv_bootstrap()
     return tlv_bootstrap_sym;
 }
 
-MachObj64::MachObj64()
-{
-    assert(I64);
-}
```

Comment:
Ah, I see, fari enough. So it's not possible to have a constructor implemented in D and call it from C++, but the default constructor works?